client/src/pages/Reports/Reports.tsx

```diff
@@ -1,19 +1,88 @@
 import './Reports.scss';
 import SectionComponent from 'components/SectionComponent/SectionComponent.tsx';
-// import Table from '@/components/Table/Table.tsx';
+import Table from 'components/Table/Table.tsx';
+import {useEffect, useState} from 'react';
+import {useNavigate} from 'react-router-dom';
+import {ManagmentServiceApiInstance, ReportSummary} from 'api/managment-service';
+import SVGIcon from 'components/SVGIcon/SVGIcon.tsx';
+import UrgencyBadge from 'components/UrgencyBadge/UrgencyBadge.tsx';
 
 const Reports = () => {
+    const [rows, setRows] = useState<ReportSummary[]>([]);
+    const [loading, setLoading] = useState<boolean>(true);
+    const navigate = useNavigate();
+
+    const handleRowClick = (id: string) => {
+        navigate(`/reports/${id}`);
+    };
+
+    const columns = [
```

Comment:
Consider annotating the `columns` with type like ~`Array<TableColumn<ReportSummary>> `